### ISSUE TITLE
[Bug] java.lang.NoSuchMethodError io.netty.handler.codec.http.HttpUtil.normalizeAndGetContentLength(Ljava/util/List;ZZ)J

### ISSUE THREAD

**Issue by langchain4j** [pinecone-io/pinecone-java-client#27]
### Is this a new bug?

- [X] I believe this is a new bug
- [X] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

Getting an exception when trying to upsert a vector:

```
Exception in thread "main" io.grpc.StatusRuntimeException: UNKNOWN
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:271)
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:252)
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:165)
	at io.pinecone.proto.VectorServiceGrpc$VectorServiceBlockingStub.upsert(VectorServiceGrpc.java:505)
	at Test.main(Test.java:42)
Caused by: java.lang.NoSuchMethodError: io.netty.handler.codec.http.HttpUtil.normalizeAndGetContentLength(Ljava/util/List;ZZ)J
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:394)
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:337)
	at io.netty.handler.codec.http2.Http2InboundFrameLogger$1.onHeadersRead(Http2InboundFrameLogger.java:56)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader$2.processFragment(DefaultHttp2FrameReader.java:476)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readHeadersFrame(DefaultHttp2FrameReader.java:484)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.processPayloadState(DefaultHttp2FrameReader.java:253)
	at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readFrame(DefaultHttp2FrameReader.java:159)
	at io.netty.handler.codec.http2.Http2InboundFrameLogger.readFrame(Http2InboundFrameLogger.java:41)
	at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder.decodeFrame(DefaultHttp2ConnectionDecoder.java:173)
	at io.netty.handler.codec.http2.Http2ConnectionHandler$FrameDecoder.decode(Http2ConnectionHandler.java:393)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.decode(Http2ConnectionHandler.java:453)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:510)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:449)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:279)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1486)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1235)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1282)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:510)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:449)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:279)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:750)
```

### Expected Behavior

Should upsert successfully without exception

### Steps To Reproduce

1. Add the following dependency:
```
<dependency>
            <groupId>io.pinecone</groupId>
            <artifactId>pinecone-client</artifactId>
            <version>0.4.0</version>
        </dependency>
```

2. Run the following code:
```java
PineconeClientConfig configuration = new PineconeClientConfig()
                .withApiKey(System.getenv("PINECONE_API_KEY"))
                .withEnvironment("asia-southeast1-gcp-free")
                .withProjectName("75dc67a");

        PineconeClient pineconeClient = new PineconeClient(configuration);

        PineconeConnectionConfig connectionConfig = new PineconeConnectionConfig()
                .withIndexName("test");

        PineconeConnection connection = pineconeClient.connect(connectionConfig);

        UpsertRequest.Builder upsertRequestBuilder = UpsertRequest.newBuilder()
                .setNamespace("default");

        List<Float> floatList = new ArrayList<Float>();
        Random random = new Random();
        for (int i = 0; i < 384; i++) {
            floatList.add(random.nextFloat());
        }

        Vector.Builder vectorBuilder = Vector.newBuilder()
                .setId(randomUUID().toString())
                .addAllValues(floatList);
        upsertRequestBuilder.addVectors(vectorBuilder.build());

        connection.getBlockingStub().upsert(upsertRequestBuilder.build());
```

### Relevant log output

```shell
Here is the output of "mvn dependency:tree -Dverbose" on a clean maven project:

[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------------< org.example:test-pinecone >----------------------
[INFO] Building Archetype - test-pinecone 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ test-pinecone ---
[INFO] org.example:test-pinecone:jar:1.0-SNAPSHOT
[INFO] \- io.pinecone:pinecone-client:jar:0.4.0:compile
[INFO]    +- io.netty:netty-tcnative-boringssl-static:jar:2.0.59.Final:compile
[INFO]    |  +- io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile
[INFO]    |  +- io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile
[INFO]    |  |  +- (io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  \- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  +- io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile
[INFO]    |  |  +- (io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  \- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  +- io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile
[INFO]    |  |  +- (io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  \- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  +- io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile
[INFO]    |  |  +- (io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |  |  \- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |  \- io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile
[INFO]    |     +- (io.netty:netty-tcnative-classes:jar:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:linux-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:osx-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:osx-aarch_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    |     +- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for cycle)
[INFO]    |     \- (io.netty:netty-tcnative-boringssl-static:jar:windows-x86_64:2.0.59.Final:compile - omitted for duplicate)
[INFO]    +- io.grpc:grpc-protobuf:jar:1.53.0:compile
[INFO]    |  +- io.grpc:grpc-api:jar:1.53.0:compile
[INFO]    |  |  +- io.grpc:grpc-context:jar:1.53.0:compile
[INFO]    |  |  +- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
[INFO]    |  |  +- com.google.errorprone:error_prone_annotations:jar:2.14.0:compile
[INFO]    |  |  \- (com.google.guava:guava:jar:31.1-android:runtime - omitted for duplicate)
[INFO]    |  +- com.google.code.findbugs:jsr305:jar:3.0.2:compile
[INFO]    |  +- com.google.protobuf:protobuf-java:jar:3.21.7:compile
[INFO]    |  +- com.google.api.grpc:proto-google-common-protos:jar:2.14.3:compile
[INFO]    |  |  \- (com.google.protobuf:protobuf-java:jar:3.21.1:compile - omitted for conflict with 3.21.7)
[INFO]    |  +- io.grpc:grpc-protobuf-lite:jar:1.53.0:compile
[INFO]    |  |  +- (io.grpc:grpc-api:jar:1.53.0:compile - omitted for duplicate)
[INFO]    |  |  +- (com.google.code.findbugs:jsr305:jar:3.0.2:runtime - omitted for duplicate)
[INFO]    |  |  \- (com.google.guava:guava:jar:31.1-android:runtime - omitted for duplicate)
[INFO]    |  \- (com.google.guava:guava:jar:31.1-android:compile - scope updated from runtime; omitted for duplicate)
[INFO]    +- io.grpc:grpc-stub:jar:1.53.0:compile
[INFO]    |  +- (io.grpc:grpc-api:jar:1.53.0:compile - omitted for duplicate)
[INFO]    |  +- com.google.guava:guava:jar:31.1-android:compile
[INFO]    |  |  +- com.google.guava:failureaccess:jar:1.0.1:compile
[INFO]    |  |  +- com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava:compile
[INFO]    |  |  +- (com.google.code.findbugs:jsr305:jar:3.0.2:compile - omitted for duplicate)
[INFO]    |  |  +- org.checkerframework:checker-qual:jar:3.12.0:compile
[INFO]    |  |  +- (com.google.errorprone:error_prone_annotations:jar:2.11.0:compile - omitted for conflict with 2.14.0)
[INFO]    |  |  \- com.google.j2objc:j2objc-annotations:jar:1.3:compile
[INFO]    |  \- (com.google.errorprone:error_prone_annotations:jar:2.14.0:compile - scope updated from runtime; omitted for duplicate)
[INFO]    +- io.grpc:grpc-netty:jar:1.53.0:compile
[INFO]    |  +- io.grpc:grpc-core:jar:1.53.0:compile
[INFO]    |  |  +- (io.grpc:grpc-api:jar:1.53.0:compile - omitted for duplicate)
[INFO]    |  |  +- com.google.code.gson:gson:jar:2.9.0:runtime
[INFO]    |  |  +- com.google.android:annotations:jar:4.1.1.4:runtime
[INFO]    |  |  +- org.codehaus.mojo:animal-sniffer-annotations:jar:1.21:runtime
[INFO]    |  |  +- (com.google.errorprone:error_prone_annotations:jar:2.14.0:runtime - omitted for duplicate)
[INFO]    |  |  +- (com.google.guava:guava:jar:31.1-android:runtime - omitted for duplicate)
[INFO]    |  |  \- (io.perfmark:perfmark-api:jar:0.25.0:runtime - omitted for duplicate)
[INFO]    |  +- io.netty:netty-codec-http2:jar:4.1.79.Final:compile
[INFO]    |  |  +- io.netty:netty-common:jar:4.1.79.Final:compile
[INFO]    |  |  +- io.netty:netty-buffer:jar:4.1.79.Final:compile
[INFO]    |  |  |  \- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  +- io.netty:netty-transport:jar:4.1.79.Final:compile
[INFO]    |  |  |  +- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- (io.netty:netty-buffer:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  \- io.netty:netty-resolver:jar:4.1.79.Final:compile
[INFO]    |  |  |     \- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  +- io.netty:netty-codec:jar:4.1.79.Final:compile
[INFO]    |  |  |  +- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- (io.netty:netty-buffer:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  \- (io.netty:netty-transport:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  +- io.netty:netty-handler:jar:4.1.48.Final:compile
[INFO]    |  |  |  +- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- (io.netty:netty-resolver:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- (io.netty:netty-buffer:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- (io.netty:netty-transport:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  +- io.netty:netty-transport-native-unix-common:jar:4.1.48.Final:compile
[INFO]    |  |  |  |  +- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  |  +- (io.netty:netty-buffer:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  |  \- (io.netty:netty-transport:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  |  \- (io.netty:netty-codec:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |  \- io.netty:netty-codec-http:jar:4.1.48.Final:compile
[INFO]    |  |     +- (io.netty:netty-common:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |     +- (io.netty:netty-buffer:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |     +- (io.netty:netty-transport:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |     +- (io.netty:netty-codec:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  |     \- (io.netty:netty-handler:jar:4.1.79.Final:compile - omitted for duplicate)
[INFO]    |  +- io.netty:netty-handler-proxy:jar:4.1.79.Final:runtime
[INFO]    |  |  +- (io.netty:netty-common:jar:4.1.79.Final:runtime - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-buffer:jar:4.1.79.Final:runtime - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-transport:jar:4.1.79.Final:runtime - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-codec:jar:4.1.79.Final:runtime - omitted for duplicate)
[INFO]    |  |  +- (io.netty:netty-codec-socks:jar:4.1.79.Final:runtime - omitted for conflict with 4.1.48.Final)
[INFO]    |  |  \- (io.netty:netty-codec-http:jar:4.1.79.Final:runtime - omitted for duplicate)
[INFO]    |  +- (com.google.guava:guava:jar:31.1-android:runtime - omitted for duplicate)
[INFO]    |  +- (com.google.errorprone:error_prone_annotations:jar:2.14.0:compile - scope updated from runtime; omitted for duplicate)
[INFO]    |  +- io.perfmark:perfmark-api:jar:0.25.0:runtime
[INFO]    |  \- (io.netty:netty-transport-native-unix-common:jar:4.1.79.Final:compile - scope updated from runtime; omitted for duplicate)
[INFO]    +- org.slf4j:slf4j-api:jar:2.0.5:runtime
[INFO]    +- (com.google.api.grpc:proto-google-common-protos:jar:2.14.3:compile - scope updated from runtime; omitted for duplicate)
[INFO]    +- org.asynchttpclient:async-http-client:jar:2.12.1:runtime
[INFO]    |  +- org.asynchttpclient:async-http-client-netty-utils:jar:2.12.1:runtime
[INFO]    |  |  +- (io.netty:netty-buffer:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (org.slf4j:slf4j-api:jar:1.7.30:runtime - omitted for conflict with 2.0.5)
[INFO]    |  |  \- (com.sun.activation:javax.activation:jar:1.2.0:runtime - omitted for duplicate)
[INFO]    |  +- (io.netty:netty-codec-http:jar:4.1.48.Final:compile - scope updated from runtime; omitted for duplicate)
[INFO]    |  +- (io.netty:netty-handler:jar:4.1.48.Final:compile - scope updated from runtime; omitted for duplicate)
[INFO]    |  +- io.netty:netty-codec-socks:jar:4.1.48.Final:runtime
[INFO]    |  |  +- (io.netty:netty-common:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-buffer:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-transport:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  \- (io.netty:netty-codec:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  +- (io.netty:netty-handler-proxy:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  +- io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.48.Final:runtime
[INFO]    |  |  +- (io.netty:netty-common:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-buffer:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-transport:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  \- (io.netty:netty-transport-native-unix-common:jar:4.1.48.Final:compile - scope updated from runtime; omitted for duplicate)
[INFO]    |  +- io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.48.Final:runtime
[INFO]    |  |  +- (io.netty:netty-common:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-buffer:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  +- (io.netty:netty-transport:jar:4.1.48.Final:runtime - omitted for conflict with 4.1.79.Final)
[INFO]    |  |  \- (io.netty:netty-transport-native-unix-common:jar:4.1.48.Final:compile - scope updated from runtime; omitted for duplicate)
[INFO]    |  +- org.reactivestreams:reactive-streams:jar:1.0.3:runtime
[INFO]    |  +- com.typesafe.netty:netty-reactive-streams:jar:2.0.4:runtime
[INFO]    |  |  +- (io.netty:netty-handler:jar:4.1.43.Final:runtime - omitted for conflict with 4.1.48.Final)
[INFO]    |  |  \- (org.reactivestreams:reactive-streams:jar:1.0.3:runtime - omitted for duplicate)
[INFO]    |  +- (org.slf4j:slf4j-api:jar:1.7.30:runtime - omitted for conflict with 2.0.5)
[INFO]    |  \- com.sun.activation:javax.activation:jar:1.2.0:runtime
[INFO]    +- com.fasterxml.jackson.core:jackson-databind:jar:2.14.2:runtime
[INFO]    |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.14.2:runtime
[INFO]    |  \- (com.fasterxml.jackson.core:jackson-core:jar:2.14.2:runtime - omitted for duplicate)
[INFO]    \- com.fasterxml.jackson.core:jackson-core:jar:2.14.2:runtime
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.917 s
[INFO] Finished at: 2023-09-28T12:30:52+02:00
[INFO] ------------------------------------------------------------------------
```

### Environment

```markdown
- **OS**: Windows 11
- **Language version**: temurin-1.8.0_372
- **Pinecone client version**: 0.4.0
```

### Additional Context

The same issue happens on version 0.3.0 as well

**Comment by rohanshah18** [comment]
Thanks for raising this issue, let me take a look and get back.

**Comment by rohanshah18** [comment]
@langchain4j I've fixed the issue, the latest released java-sdk [v0.5.1](https://github.com/pinecone-io/pinecone-java-client/releases/tag/v0.5.1) should've the updates.
